src/raftstore/store/peer.rs

```diff
@@ -370,14 +370,6 @@ impl Peer {
     pub fn activate<T, C>(&self, ctx: &PollContext<T, C>) {
```

Comment:
please update the comment, and I prefer writting 
```
let mut meta = ctx.store_meta.lock().unwrap();
            meta.readers
                .insert(self.region_id, ReadDelegate::from_peer(self));
```
here rather than in `post_raft_ready_append`

Reply:
I guess seperating them is better for now, because `activate` is also called in `fsm/peer.rs`, where the meta is already locked. It will introduce much more changes.

Reply:
Oh! so in `fsm/peer.rs` splitting a new peer, it doesn't register the read delegate?